README.md

```diff
@@ -111,12 +111,17 @@ If you are interested in any of the following
 
 * Generation of parallel code (CPU, GPU, multi-node via MPI);
 * Performance tuning;
-* Benchmarking Devito;
-* Any other aspect concerning application performance;
+* Benchmarking operators;
```

Comment:
We may wanna add a small section or link to the env variables, lots of people complaining thay can't find how to set openmp/compiler/....

Reply:
they are in the FAQ now

Reply:
People don't look that far or don't find it (there is a lot of tabs in the Wiki) and and easy access to it is really necessary. Can guarantee you I get people complaining very often they can't find how to set it

Reply:
FAQ now linked from here: https://www.devitoproject.org/ and from this README too under Resources.

Two bullets above: "Generation of parallel code (CPU, GPU, multi-node via MPI);" -> go-to `benchmarks/user/README.md` (where all relevant env vars are described)

Do you really think linking the FAQ **also** from here would make such a big difference?  IMHO unnecessary, but I can do that if you're sure that this is the central point of the README

Reply:
I think it's fine, not trying to create extra work. People will always find way to complain anyway.